concent_api/core/tests/test_payments_sci_backend.py

```diff
@@ -57,7 +57,7 @@ def test_that_if_number_of_blocks_from_timestamp_is_smaller_than_required_confs_
 
         self.assertEqual(list_of_payments, [])
 
-    def test_that_sci_backend_get_list_of_payments_should_return_list_of_forced_subtask_payments(self):
+    def test_that_sci_backend_get_list_of_payments_should_return_list_of_forced_payments(self):
```

Comment:
"forced" in the test name, `SETTLEMENT` in the test's body...

Reply:
Let's settle on calling them settlement payments :)